### ISSUE TITLE
Test suite hangs (with newer CPAN.pm?)

### ISSUE THREAD

**Issue by eserte** [cpan-testers/CPAN-Reporter-Smoker#13]
It seems that the test suite hangs if newer CPAN.pm is installed. See http://www.cpantesters.org/cpan/report/86769c02-834d-11ec-94c9-0cbb56ab8115 for a sample test report.

Statistical analysis suggests that there a change in CPAN.pm 2.29 is causing the problem:

```
****************************************************************
Regression 'mod:CPAN'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	37008030521244192.00
[1='eq_1.9600_01']	      0.0000	      0.0000	   7.13
[2='eq_1.9800']	      0.0000	      0.0000	   7.10
[3='eq_2.00']  	      0.0000	      0.0000	   8.34
[4='eq_2.05']  	      0.0000	      0.0000	   7.38
[5='eq_2.11']  	      0.0000	      0.0000	   9.47
[6='eq_2.11_01']	      0.0000	      0.0000	  10.96
[7='eq_2.14']  	      0.0000	      0.0000	  10.29
[8='eq_2.16']  	      0.0000	      0.0000	  10.49
[9='eq_2.17']  	      0.0000	      0.0000	   9.20
[10='eq_2.18'] 	      0.0000	      0.0000	   9.91
[11='eq_2.20'] 	      0.0000	      0.0000	  11.48
[12='eq_2.21'] 	      0.0000	      0.0000	   6.72
[13='eq_2.22'] 	      0.0000	      0.0000	  10.69
[14='eq_2.26'] 	      0.0000	      0.0000	  11.89
[15='eq_2.27'] 	      0.0000	      0.0000	   9.48
[16='eq_2.28'] 	      0.0000	      0.0000	   2.18
[17='eq_2.29'] 	     -1.0000	      0.0000	-32049894575427328.00
[18='eq_2.33'] 	     -1.0000	      0.0000	-32049894575427324.00
[19='eq_2.34'] 	     -1.0000	      0.0000	-33100988783555596.00
[20='eq_2.36'] 	     -1.0000	      0.0000	-26168629339930492.00

R^2= 1.000, N= 228, K= 21
****************************************************************
```

**Comment by DSolimano** [comment]
I think I got to the bottom of this. It's related to the pushy_https change and the associated change in the CHECKSUMS file, so we have two problems with modern CPAN.pm, the first is it ignores the custom url list unless pushy_https is set to 0, and the second is that the script that calculates the checksums needs to be updated and rerun.

For the first, I edited `t/data/MyConfig.pm` to have a `'pushy_https' => q[0]` in it

For the second I updated `/t/CPAN/update-checksums.pl`,  `updatedir( $File::Find::name );` got changed to `updatedir( $File::Find::name, 'authors/id/');` then reran it.
